setup.py

```diff
@@ -49,8 +49,15 @@ class BuildProtoCommand(Command):
                   ('outdir=', 'o', 'Where to output .py files.')]
 
   def initialize_options(self):
-    self.protoc = '/usr/bin/protoc'
-    self.protodir = '/usr/include'
+    try:
+      prefix = subprocess.check_output(
+          'pkg-config --variable prefix protobuf'.split()).strip()
+    except subprocess.CalledProcessError:
+      # Default to /usr?
+      prefix = '/usr'
```

Comment:
at least check that the system is *nix when applying this default.  for completeness, it would be nice to provide a default on Windows, but you can just raise for now.

the requirement right now isn't to fully support windows, but rather that when run on windows, failures should be obvious (like a raise here), not obscured behind things like platform-specific path defaults and path separators.  that said, windows support is a fairly high priority after the frontend.